rasa/core/channels/mattermost.py

```diff
@@ -16,12 +16,22 @@ class MattermostBot(MattermostAPI, OutputChannel):
     def name(cls):
         return "mattermost"
 
-    def __init__(self, url, team, user, pw, bot_channel):
+    @classmethod
+    def from_credentials(cls, credentials):
+        if not credentials:
+            cls.raise_missing_credentials_exception()
+
+        return cls(
+            credentials.get("webhook_url"),
```

Comment:
Why not also get the team, user, pw, and url from the credentials?

Reply:
Only the webhook_url is necessary, user, pw etc. is not needed for attachments.